testsuite/fuzzer/fuzz/fuzz_targets/move/bytecode_verifier_compiled_modules.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) The Move Contributors
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+#![no_main]
+use libfuzzer_sys::{fuzz_target, Corpus};
+use move_binary_format::errors::VMError;
+use move_bytecode_verifier::VerifierConfig;
+use move_core_types::vm_status::StatusType;
+mod utils;
+use utils::vm::RunnableState;
+
+fn check_for_invariant_violation_vmerror(e: VMError) {
+    if e.status_type() == StatusType::InvariantViolation
+        // ignore known false positive
+        && !e
+            .message()
+            .is_some_and(|m| m.starts_with("too many type parameters/arguments in the program"))
```

Comment:
Hm, where does this come from?

Update: checked the code, why bytecode verifier returns an invariant violation.... 🤦‍♂️ we should probably fix this, coupling with some other verifier feature...